js/src/dropdown.js

```diff
@@ -1,3 +1,5 @@
+/* global Popper */
```

Comment:
Currently they are no global solution for that issue (see : https://github.com/twbs/bootstrap/issues/17201#issuecomment-274243781).
So for now I'll pass on that.

Maybe @bardiharborow have some feedbacks ?